src/matrix.js

```diff
@@ -1417,6 +1417,31 @@ export class AbstractMatrix {
     return this;
   }
 
+  /**
+   * Center column by column of a matrix
+   * @param {Matrix} matrix - Column index
+   * @return {Matrix}
+   */
+  centerColumns() {
```

Comment:
Can you please implement it like the other statistical-related functions (mean, variance, standardDeviation)? It means calling the method `center(by)` with the `by` parameter that can be either:
- `undefined`: treat the matrix as a whole
- `'column'`: center the columns
- `'row'`: center the rows